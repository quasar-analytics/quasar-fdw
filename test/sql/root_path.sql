/* move the path to the table instead of the server */
CREATE SERVER quasar_root FOREIGN DATA WRAPPER quasar_fdw
       OPTIONS (server 'http://localhost:8080'
               ,path '/');

CREATE FOREIGN TABLE zips_root(city varchar, pop integer, state char(2))
       SERVER quasar_root OPTIONS (table 'local/quasar/zips');

SELECT * FROM zips_root ORDER BY city,pop LIMIT 5;
