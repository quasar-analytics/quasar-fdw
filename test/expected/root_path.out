/* move the path to the table instead of the server */
CREATE SERVER quasar_root FOREIGN DATA WRAPPER quasar_fdw
       OPTIONS (server 'http://localhost:8080'
               ,path '/');
CREATE FOREIGN TABLE zips_root(city varchar, pop integer, state char(2))
       SERVER quasar_root OPTIONS (table 'local/quasar/zips');
SELECT * FROM zips_root ORDER BY city,pop LIMIT 5;
    city    |  pop  | state 
------------+-------+-------
 AARON      |   270 | KY
 AARONSBURG |   100 | PA
 ABAC       | 27906 | GA
 ABBEVILLE  |  1991 | GA
 ABBEVILLE  |  4649 | MS
(5 rows)

